normandy/classifier/templates/admin/classifier_preview.html

```diff
@@ -0,0 +1,78 @@
+{% extends 'admin/base_site.html' %}
+{% load static %}
+
+{% block extrastyle %}
+  {{ block.super }}
+  <link rel="stylesheet" type="text/css" href="{% static 'admin/css/forms.css' %}" />
+  <link rel="stylesheet" type="text/css" href="{% static 'admin/css/classifier_preview.css' %}" />
+  {{ form.media.css }}
+{% endblock %}
+
+{% block extrahead %}
+  {{ block.super }}
+  <script type="text/javascript" src="{% static 'admin/js/core.js' %}"></script>
+  <script type="text/javascript" src="{% static 'admin/js/vendor/jquery/jquery.js' %}"></script>
+  <script type="text/javascript" src="{% static 'admin/js/jquery.init.js' %}"></script>
+  <script type="text/javascript" src="{% url 'admin:jsi18n' %}"></script>
+  {{ form.media.js }}
+{% endblock %}
+
+{% block content %}
+  <form method="get">
+    {{ form.non_field_errors }}
+    <div class="client-properties">
+      <div class="field">
+        {{ form.locale.errors }}
+        {{ form.locale.label_tag }}
+        {{ form.locale }}
+      </div>
+      <div class="field">
+        {{ form.country.errors }}
+        {{ form.country.label_tag }}
+        {{ form.country }}
+      </div>
+      <div class="field">
+        {{ form.release_channel.errors }}
+        {{ form.release_channel.label_tag }}
+        {{ form.release_channel }}
+      </div>
+      <div class="field">
+        {{ form.request_time.errors }}
+        {{ form.request_time.label_tag }}
+        {{ form.request_time }}
+      </div>
+    </div>
+    <div class="submit-row">
+      <input class="default" type="submit" value="Match Recipes"></input>
+    </div>
+  </form>
+  {% if bundle %}
```

Comment:
I think this should always include a list of matched recipes (if there is input in the above form), and show "No recipes matched" if there aren't any.